src/pages/Components/DadosCandidato.tsx

```diff
@@ -0,0 +1,41 @@
+import { List, ListItem, ListItemText, Link, Typography } from "@mui/material";
+import { IAluno } from "../Revisao/Interfaces";
+
+interface IProps {
+  dadosInscrito: IAluno | undefined;
+}
+
+function DadosCandidato({ dadosInscrito }: IProps) {
+  return (
+    <List
+      component="nav"
+      aria-labelledby="dados-inscrito"
+      subheader={
+        <Typography variant="h6" sx={{ mt: 1 }}>
+          Dados do Candidato
+        </Typography>
+      }
+    >
+      {dadosInscrito && (
+        <>
+          <ListItem>
+            <ListItemText primary={dadosInscrito.nome} />
```

Comment:
```suggestion
            <ListItemText primary={dadosInscrito?.nome} />
```

Como pode ser undefined, melhor botar a ? 
Obs.: Era pra o lint ter reclamado disso

Reply:
acho q n reclamou pq da condicional na linha 19

Reply:
Aaah, verdade